frontend/src/App.vue

```diff
@@ -61,9 +61,29 @@ export default {
       const itemFqid = contextualItem?.item?.fqid;
       await this.rpc.invoke("performAction", [itemFqid, index, contextualItem.context]);
     },
+
+    getItemByFqid(fqid) {
+      let items = [];
+      this.collections.forEach(col => {
+        items = items.concat(col.items);
+      });
+      const flatten = [];
+      let itemnode;
+      while (items.length > 0) {
```

Comment:
I think we can do another iteration to improve the maintainability of this function.
But we can do it tomorrow after release because you are short on time...